connector-google-chat/src/main/java/com/redhat/cloud/notifications/connector/google/chat/GoogleChatExceptionProcessor.java

```diff
@@ -14,16 +14,17 @@ public class GoogleChatExceptionProcessor extends ExceptionProcessor {
             "with status code [%d] and body [%s]";
 
     @Override
-    protected void log(Throwable t, Exchange exchange) {
+    protected void process(Throwable t, Exchange exchange) {
         if (t instanceof HttpOperationFailedException) {
+            HttpOperationFailedException e = (HttpOperationFailedException) t;
```

Comment:
We can directly use synthaxe `if (t instanceof HttpOperationFailedException e)` to initialize e

Reply:
Thanks! My brain is still stuck with Java 11 😄